chains.yaml

```diff
@@ -888,3 +888,15 @@
     - build/wasmd
   build-env:
     - BUILD_TAGS=muslc
+
+# Avalanche
+- name: avalanche
+  github-organization: ava-labs
+  github-repo: avalanchego
+  dockerfile: avalanche
+  build-target: 
+    bash scripts/build.sh
+  binaries:
+    - build/avalanchego
+  platforms:
```

Comment:
I tested the `linux/arm64` build on our runner and it works! So we can remove the platforms from avalanche